plugins/destination.go

```diff
@@ -2,21 +2,23 @@ package plugins
 
 import (
 	"context"
-	"fmt"
 	"time"
 
+	"github.com/cloudquery/plugin-sdk/cqtypes"
 	"github.com/cloudquery/plugin-sdk/schema"
 	"github.com/cloudquery/plugin-sdk/specs"
 	"github.com/rs/zerolog"
+	"golang.org/x/sync/errgroup"
 )
 
 type NewDestinationClientFunc func(context.Context, zerolog.Logger, specs.Destination) (DestinationClient, error)
 
 type DestinationClient interface {
 	Migrate(ctx context.Context, tables schema.Tables) error
-	Write(ctx context.Context, table string, data map[string]interface{}) error
+	Read(ctx context.Context, table *schema.Table, sourceName string, res chan<- *schema.DestinationResource) error
```

Comment:
Where do we use `Read`?

Reply:
This is very important for testing otherwise it produces a lot of boilerplate code in destination plugins. This way we can test round-trip values and data consistency. https://github.com/cloudquery/plugin-sdk/pull/320/files#diff-e6872d0aa4c7b0a55579fa1fd579f89364e68173611f8bf7d574f95c9d8cc65c